app/models/category.rb

```diff
@@ -73,6 +73,7 @@ class Category < ActiveRecord::Base
   after_save :publish_discourse_stylesheet
   after_save :publish_category
   after_save :reset_topic_ids_cache
+  after_save :clear_subcategory_ids_cache
```

Comment:
It would be ideal to clear the cache only for new records or if `parent_category_id` changed, but I wanted to play it safe for now.